arbeitszeit/records.py

```diff
@@ -129,6 +129,19 @@ class CoordinationTenure:
     start_date: datetime
 
 
+@dataclass
+class CoordinationTransferRequest:
+    id: UUID
+    requesting_coordination_tenure: UUID
+    candidate: UUID
+    request_date: datetime
+    transfer_date: Optional[datetime]
+
+    @property
```

Comment:
I just read some documentation on the \@property decorator — but I am still wondering why you are using this decorator for is_open.  Is that so that you can write transferRequest.is_open instead of transferRequest.is_open()?

Reply:
In terns of clean code, I think it would be better to use the function directly rather than to hide it as a property.  In efficiency terms, the implementation of @property also adds tiny but unnecessary overhead.